scripts/consume_sitemaps.py

```diff
@@ -11,18 +14,22 @@
 import isb_web
 import isb_web.config
 import logging
+import re
+
+from isb_lib.models.thing import Thing
 from isb_lib.sitemaps.sitemap_fetcher import (
     SitemapIndexFetcher,
     SitemapFileFetcher,
-    ThingFetcher,
+    ThingFetcher, ThingsFetcher,
 )
 from isb_web import sqlmodel_database
-from isb_web.sqlmodel_database import SQLModelDAO
+from isb_web.sqlmodel_database import SQLModelDAO, all_thing_identifiers, thing_identifiers_from_resolved_content
 
-CONCURRENT_DOWNLOADS = 50
+CONCURRENT_DOWNLOADS = 50000
```

Comment:
It ended up being that fetching everything in one sitemap file was the most performant and didn't seem to stress my machine too much.  We might end up tweaking this on the cyverse machines depending on how it looks out there.